code/game/objects/items/devices/traitordevices.dm

```diff
@@ -248,6 +248,27 @@ effective or pretty fucking useless.
 	active = !active
 	if(active)
 		GLOB.active_jammers |= src
+		if(iscyborg(user))
+			START_PROCESSING(SSobj,src)
 	else
 		GLOB.active_jammers -= src
+		STOP_PROCESSING(SSobj,src)
 	update_icon()
+
+/obj/item/jammer/process()
+	var/mob/living/silicon/robot/R = loc
+	if(!R || !iscyborg(R))
```

Comment:
```suggestion
	if(QDELETED(R) || !iscyborg(R))
```
Might want to make a `/obj/item/jammer/cyborg` subtype instead, keeping a reference to the cyborg.

Reply:
wait hold on

Why wouldn't !R work there?

Reply:
It would, just not for things in the process of being deleted.